scan/src/components/oracle-script/OracleScriptBridgeCode.re

```diff
@@ -153,21 +153,27 @@ module LanguageIcon = {
 let getFileNameFromLanguage = (~language, ~dataType) => {
   let dataTypeString = dataType |> Obi.dataTypeToString;
   switch (language) {
-  | Solidity => {j|$(dataTypeString)Decoder.sol|j}
+  | Solidity => "Decoders.sol"
   | Go => {j|$(dataTypeString)Decoder.go|j}
   };
 };
 
 let getCodeFromSchema = (~schema, ~language, ~dataType) => {
   switch (language) {
-  | Solidity => Obi.generateDecoderSolidity(schema, dataType)
+  | Solidity => Obi.generateDecoderSolidity(schema)
   | Go => Obi.generateDecoderGo("main", schema, dataType)
```

Comment:
Could we remove `dataType` on Go too?

Reply:
We do not have ParamsDecoder for Go yet. And I believe we agreed to leave it as is for now until IBC is available.